styles.css

```diff
@@ -0,0 +1,73 @@
+body{
+background-image:url("https://assets.bitdegree.org/online-learning-platforms/storage/media/2018/08/what-is-a-web-developer.jpg");
+/* text-align:center; */
+margin:0 auto;
+font-size:20;
+font-family:sans-serif;
+}
+hr{
+border-color:black;
+margin-bottom:20;
+width:80%;
+}
+p{
+font-size:20px;
+}
+h1{
+margin-top:50;
+}
+h1,p{
+color:white;
+text-align:center;
+
+}
+form{
```

Comment:
please center the form so it can match the design in the reference. Matching the design helps clients to be happy with your work and rely on your work for their supply.